src/fides/api/ctl/routes/system.py

```diff
@@ -0,0 +1,211 @@
+from typing import List
+
+from fastapi import Depends, HTTPException, Security
+from fastapi_pagination import Page, Params
+from fastapi_pagination.bases import AbstractPage
+from fastapi_pagination.ext.sqlalchemy import paginate
+from loguru import logger
+from pydantic.types import conlist
+from sqlalchemy.orm import Session
+from starlette.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_404_NOT_FOUND,
+    HTTP_422_UNPROCESSABLE_ENTITY,
+)
+
+from fides.api.ctl.sql_models import System  # type: ignore[attr-defined]
+from fides.api.ctl.utils.api_router import APIRouter
+from fides.api.ops.api import deps
+from fides.api.ops.api.v1.endpoints.connection_endpoints import (
+    requeue_requires_input_requests,
+    validate_secrets,
+)
+from fides.api.ops.api.v1.scope_registry import (
+    CONNECTION_CREATE_OR_UPDATE,
+    CONNECTION_READ,
+)
+from fides.api.ops.api.v1.urn_registry import (
+    CONNECTION_TYPES,
+    SYSTEM_CONNECTIONS,
+    V1_URL_PREFIX,
+)
+from fides.api.ops.models.connectionconfig import ConnectionConfig
+from fides.api.ops.schemas.api import BulkUpdateFailed
+from fides.api.ops.schemas.connection_configuration.connection_config import (
+    BulkPutConnectionConfiguration,
+    ConnectionConfigurationResponse,
+    CreateConnectionConfigurationWithSecrets,
+    SaasConnectionTemplateValues,
+)
+from fides.api.ops.service.connectors.saas.connector_registry_service import (
+    create_connection_config_from_template_no_save,
+    load_registry,
+    registry_file,
+)
+from fides.api.ops.util.oauth_util import verify_oauth_client
+from fides.lib.exceptions import KeyOrNameAlreadyExists
+
+router = APIRouter(tags=["System"], prefix=f"{V1_URL_PREFIX}{SYSTEM_CONNECTIONS}")
```

Comment:
These new routes still show up under the same `System` section in the swagger docs despite being attached to a completely separate router.

Reply:
AFAIK yes. I tried adding them to the original router from the crud file and that sort of worked but it was a little buggy and complicated. I tried this second and worked first try. I guess the swagger docs code is smart enough to group routes with the same tags together